model/src/main/kotlin/utils/DatabaseUtils.kt

```diff
@@ -42,10 +42,6 @@ object DatabaseUtils {
             "URL for PostgreSQL storage is missing."
         }
 
-        require(config.schema.isNotBlank()) {
-            "Schema for PostgreSQL storage is missing."
-        }
```

Comment:
The schema could be set to a blank string in the configuration file so this check is still valid.